android/Dockerfile

```diff
@@ -1,3 +1,13 @@
+#single point for ARG definitions
+ARG ANDROID_TOOLS=8092744
+ARG ANDROID_LEVEL=31
+ARG ANDROID_LEVEL_TOOLCHAIN=31
+ARG ANDROID_BUILD_TOOLS=31.0.0
+ARG NDK=23.1.7779620
+ARG JOBS=4
```

Comment:
The default JOBS argument should match the environment where we plan to automatically build the images. Previously this was Dockerhub although that was never implemented.

Reply:
Previously hardcoded value was 2, increased it a bit for my local machine. Generally the whole script is far from being done. I am not a docker expert (1 day of experience :) ) but it seems like a lot of e.g. caching could be introduced (e.g. qt builds). I would leave it like that.

Reply:
You can increase it locally by passing --build-arg "JOB=X" on the command line without modifying the dockerfile itself

Reply:
Ok, did not investigate that. I will restore its value to 2.